changelog/unreleased/3143

```diff
@@ -0,0 +1,8 @@
+Enhancement: Move to AndroidX Preference and new structure for settings
+
+Settings have been updated to use the current Android's recommendation, AndroidX framework.
+In addition, they have been reorganized into subsections for a better understanding and
+navigation structure. Also, new features have been added: now, source path and behaviour
+in auto uploads can be chosen differently for pictures and videos.
+
+https://github.com/owncloud/android/issues/2867 https://github.com/owncloud/android/pull/3143
```

Comment:
Should the links be placed in different lines?

Reply:
I think it would be better, but taking a look at the [template](https://github.com/owncloud/android/blob/feature/settings_androidx/changelog/TEMPLATE.md) and some other files in the [unreleased folder](https://github.com/owncloud/android/tree/feature/settings_androidx/changelog/unreleased) I found that most of them follow this structure

Reply:
ok, imho it will be more structured in different lines, but, as you said, we have to change more items... will do in a separate PR in that case